README.md

```diff
@@ -13,12 +13,35 @@ The MLPerf Training rules are available at [training\_rules](https://github.com/
 
 The MLPerf HPC specific rules are at [hpc\_training\_rules](https://github.com/mlcommons/training_policies/blob/master/hpc_training_rules.adoc).
 
+## Compliance
+The MLPerf logging package implements logging and compliance-checking utilities
+for MLPerf benchmarks in hpc-1.1.0 branch of the MLPerf logging repository (https://github.com/mlcommons/logging/tree/hpc-1.0-branch).
+These work for the HPC v2.0 submissions as well.
+
+To install and test compliance of your runs/submissions:
+
+```
+# Install the package into your python environment.
+# A development install (-e) is recommended for now so you can pull new updates.
+git clone -b hpc-1.0-branch https://github.com/mlperf-hpc/logging.git mlperf-logging
```

Comment:
that's the old v0.7 URL. Should be updated to use https://github.com/mlcommons/logging

Reply:
Fixed in 7cf95be6253ad51b5d9b96b78f8757067e50331b